evm-ext/src/native_impl.rs

```diff
@@ -165,7 +171,7 @@ fn evm_transact_inner(
 
     // Commit the changes to the DB so that future Move transactions using
     // the same session will see them.
-    evm_native_ctx.db.commit(outcome.state);
+    db.commit(outcome.state);
 
```

Comment:
Correct me if I'm wrong, but do we still need to commit changes to the db at the end? As now it's not part of context, it's not persisted across invocations of this fn. And from what I see in `commit()` source code, it doesn't mutate the actual db that was passed into it (and thus storage tree or resolver with it), but only modifies its inner mappings for accounts and contracts.

Reply:
You're right, that has no effect now.